test/io/test_io.py

```diff
@@ -1095,3 +1095,93 @@ def test_fail_with_automatic_recovery_disabled_and_terminated_using_query(defaul
 
         exitCode = wait_until_exit(postgrest)
         assert exitCode == 1
+
+
+def test_server_timing_jwt_should_decrease_on_subsequent_requests(defaultenv):
+    "assert that server-timing duration for JWT should decrease on subsequent requests"
+
+    env = {
+        **defaultenv,
+        "PGRST_DB_PLAN_ENABLED": "true",
+        "PGRST_JWT_CACHE_MAX_LIFETIME": "86400",
+        "PGRST_JWT_SECRET": "@/dev/stdin",
+        "PGRST_DB_CONFIG": "false",
+    }
+
+    headers = jwtauthheader(
+        {
+            "role": "postgrest_test_author",
+            "exp": int(
+                (datetime.now(timezone.utc) + timedelta(minutes=30)).timestamp()
+            ),
+        },
+        SECRET,
+    )
```

Comment:
@steve-chavez I am adding `exp` here because we don't have one by default as far as I understand. However, in other tests where I didn't add any `exp` (see other test cases below in `test_io.py`) the following code that should execute never gets executed:
```haskell
expireJSON <- lookup "exp" (authClaims res)
case expireJSON of
  JSON.Number seconds <- Just $ CL.TimeSpec ...
  _                   <- Just $ CL.TimeSpec ... -- this never gets executed

```
Somehow, it is always finding an `exp` key in `authClaims`. Do we have an `exp` by default in our tokens?